src/app/(guest)/exams/[id]/answer/[mode]/page.tsx

```diff
@@ -100,12 +100,12 @@ const Exam: React.FC<ExamPageProps> = ({ params }) => {
   }, [params.id, params.mode, router, setQuestions]);
 
   return (
-    <section className="h-[88vh] flex flex-col items-center overflow-x-scroll">
-      <p className="text-xl font-bold uppercase mt-10 ml-5 text-center px-4">
+    <section className="flex flex-col items-center overflow-x-scroll">
+      <p className="px-4 mt-10 ml-5 text-xl font-bold text-center uppercase">
         Exame de{' '}
         <span className="text-primary">{subject ? subject : <Skeleton width={100} />}</span>
       </p>
-      <div className="mb-12 w-screen">
+      <div className="w-screen mb-12">
         {questions[0] ? (
           <ExamNumerationContainer>
```

Comment:
The exam numeration background is not changed in dark mode & needs tinkering with the answered styling for contrast purposes.

Reply:
I agree with you, but for now I want to keep this way, to not confuse the users.
Because we had this discussion on the issue #45.